src/python/pants/backend/python/rules/pex.py

```diff
@@ -56,6 +57,20 @@ def create_from_adaptors(
         return PexRequirements(all_target_requirements)
 
 
+Spec = Tuple[str, str]  # e.g. (">=", "3.6")
```

Comment:
Maybe I'm missing something, but do we actually need to parse the spec out into two components?

Reply:
No, we don't, but this is what `Requirement.parse` does by default so I rolled with it. 

I do like that having them separated allows us to sort by version number rather than lexigraphic sorting. To me, it's easier to read `>=2.7,==2.7.5,>=3.6,==3.7.8` than `==2.7.5,==3.7.8,>=2.7,>=3.6`.